File: GRLTest/MultirootmodeL/MultirootmodeL.js
function MultirootmodeL( J,G){
	expr = 0.25*Math.max(0, Math.min(100.0, -0.25*G, 0.25*J)) + 0.75*Math.max(0, Math.min(100.0, 0.25*G, 0.75*J));
	return expr;
}

File: GRLTest/MultirootmodeL/MultirootmodeL.cpp
#include <iostream>
using namespace std;

double MultirootmodeL( double J,double G){
	double expr = 0.25*std::max(0, std::min(100.0, std::min(-0.25*G, 0.25*J))) + 0.75*std::max(0, std::min(100.0, std::min(0.25*G, 0.75*J)));
	return expr;
}
